logging/v5_7/logging-5-7-getting-started.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: ASSEMBLY
+
+[id="logging-getting-started-5-7"]
+= Getting started with logging 5.7
+
+This overview of the logging deployment process is provided for ease of reference. It is not a substitute for full documentation. For new installations, *Vector* and *LokiStack* are recommended.
++
+--
+include::snippets/logging-under-construction-snip.adoc[]
+--
+
+--
+include::snippets/logging-compatibility-snip.adoc[]
+--
+
+.Prerequisites
+* LogStore preference: *Elasticsearch* or *LokiStack*
+* Collector implementation preference: *Fluentd* or *Vector*
+* Credentials for your log forwarding outputs
+
+.Procedure
+--
+include::snippets/logging-elastic-dep-snip.adoc[]
+--
+. Install the Operator for the logstore you'd like to use.
+** For *Elasticsearch*, install the *OpenShift Elasticsearch Operator*.
+** For *LokiStack*, install the *Loki Operator*.
+*** Create a `LokiStack` custom resource (CR) instance.
+
+. Install the *Red Hat OpenShift Logging Operator*.
```

Comment:
```suggestion
. Install the *Red Hat OpenShift Logging* Operator.
```
The complete name of the operator is just *Red Hat OpenShift Logging* or *Red Hat OpenShift Logging Operator*. Please make the change accordingly.